Tests/GitHubRepositories.tests.ps1

```diff
@@ -662,7 +662,7 @@ try
                     OwnerName = 'octocat'
                     RepositoryName = 'IncorrectRepostioryName'
                 }
-                { Get-GitHubRepositoryVulnerabilityAlert @getGitHubRepositoryVulnerabilityAlertParms } |
+                { Test-GitHubRepositoryVulnerabilityAlert @getGitHubRepositoryVulnerabilityAlertParms } |
```

Comment:
I still contend that a `Test-*` function should never throw.

Reply:
See response above.